libr/asm/arch/include/libiberty.h

```diff
@@ -308,7 +308,12 @@ extern double physmem_available (void);
    as new/delete and new[]/delete[].  */
 
 /* Scalar allocators.  */
-
+#ifdef _MSC_VER
+#include <stdlib.h>
```

Comment:
Linux doesnt have any of those xmalloc, xcalloc.. why is this necessary on windows?